pkg/component/autoscaling/clusterautoscaler/cluster_autoscaler_test.go

```diff
@@ -595,7 +597,7 @@ subjects:
 		By("Create secrets managed outside of this package for whose secretsmanager.Get() will be called")
 		Expect(fakeClient.Create(ctx, &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: "generic-token-kubeconfig", Namespace: namespace}})).To(Succeed())
 
-		clusterAutoscaler = New(c, namespace, sm, image, replicas, nil, nil)
+		clusterAutoscaler = New(c, namespace, sm, image, replicas, nil, 0, nil)
```

Comment:
Nit: Maybe try with a value different than `0` (since `0` is the null value of `int64`).